Toronto Star.js

```diff
@@ -56,10 +55,10 @@ function detectWeb(doc, url) {
 function getSearchResults(doc, checkOnly) {
 	var items = {};
 	var found = false;
-	var rows = ZU.xpath(doc, '//a[span[contains(@class, "story__headline")]]');
-	for (var i = 0; i < rows.length; i++) {
-		var href = rows[i].href;
-		var title = ZU.trimInternal(rows[i].textContent);
+	var rows = doc.querySelectorAll('a[data-test-id="mediacard"]');
```

Comment:
It looks like `'.c-search-results-item > a'` would be a better selector here. I'm always a little skeptical that these `data-test-id` attributes will stick around, since they're clearly meant to be internal.

Reply:
Hmm, I see. I didn't realize the selector you had would work on other pages. That is worth keeping, then.

Reply:
I realized that the selector I had still missed some articles (see `.c-feature-mediacard` on [this page](https://www.thestar.com/topic.ottawa.html)). The new selector (`a[class*="-mediacard"]`) seems to be pretty consistent and also avoids the `data-test-id` attribute